api/provision/result-fn.test.ts

```diff
@@ -0,0 +1,77 @@
+import { Context } from "aws-lambda";
+import { handler } from "./result-fn";
+import { mockClient } from "aws-sdk-client-mock";
+import { sqsClient } from "../../utils/aws-sdk/sqs";
+import { ValidationErrorResponse } from "../../utils/response";
+import { match } from "sinon";
```

Comment:
Was this supposed to be used?

Reply:
woops, fixed

Reply:
perhaps a git push is needed?

Reply:
yeah sorry, i tend to say that a thing is fixed the moment i change it locally and in this case got mega sidetracked...  :)